randovania/games/prime1/layout/preset_describer.py

```diff
@@ -143,7 +143,7 @@ def different_xyz_randomization(diff_xyz):
                     },
                 ),
                 {
-                    "Progressive suit damage reduction": configuration.progressive_damage_reduction,
+                    "Damage reduction": configuration.damage_reduction,
```

Comment:
All this does is make "Damage reduction" always appear in the description. The value on the right is a boolean that controls if the string in the left shows up or not.

Reply:
What's the appropriate behavior? Show it only if "Default" is not selected, or to have separate options for "Progressive"/"Additive"?

Reply:
what I would do:
if default: dont show
if progressive/additive: show "Progressive" / "Additive".

Reply:
I would always show the reduction type. I would only hide `Default` if it was randovania's default, but it is not

Reply:
hm... this makes me want to rename "default" into something else. "Maximum"? "Best Suit"?